parser/statementStartTokens.coffee

```diff
@@ -0,0 +1,17 @@
+class StartTokens
+  @statement = [
+    'for', 'while',                                  # Start of statement
+    @expression
```

Comment:
Oh, okay. I reverted back to the old `at` implementation (as seen in the iki compiler) since I was getting errors otherwise.